app/Services/BulkService.php

```diff
@@ -415,6 +415,14 @@ protected function updateRelations(array $filledFields)
                 $this->count++;
                 $this->total++;
             }
+            if (request()->unmirror && $relation->mirror) {
```

Comment:
Using `request()` means we can't ever *unit test* this class (since request() is only available when executing the whole framework and not when testing just a small feature).

Instead, in `editing()`, just before calling `$this->updateRelations()`, check the `$fields` array for the options to build an array as a second parameter for the `updateRelations()` call.